src/main.js

```diff
@@ -148,6 +155,29 @@ app.get('/screenshot/:url(*)', async(request, response) => {
   }
 });
 
+app.get('/pdf/:url(*)', async(request, response) => {
+  if (isRestricted(request.params.url)) {
+    response.status(403).send('Render request forbidden, domain excluded');
+    return;
+  }
+
+  try {
+    const start = now();
+    const result = await renderer.printToPDF(request.params.url, request.query, config);
```

Comment:
I assume this is the line that could throw in practice? I usually try to scope the try/catch to the smallest possible line, otherwise you could be hiding exceptions from dumb bugs.

Reply:
I see. I'm gonna keep it as is for now, since they're all like that so it should be changed consistently.